### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by alex94945**
@alexneigher

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -46,6 +45,8 @@
     resources :users
   end
 
+  get 'contact', to: 'contact#show', as: 'contact'
```
Comment (alexneigher) [inline config/routes.rb]: 2 options here. 
1) this looks fine, 🚢 it
2) Do something more generic like `get "/pages/:page" => "static_pages#show"`
  - and then have a show action in the static-pages controller that looks like:
  ```
     def show
      render template: "pages/#{params[:page]}"
     end
   ```

/shrug

File: config/routes.rb [hunk 2]
```diff
@@ -28,7 +28,6 @@
   resources :categories
   resources :companies
 
-  resources :budgets
```
Comment (alexneigher) [inline config/routes.rb]: are you planning on getting rid of the models and the database tables for this as well? I think there may be some lingering views as well

Comment (alex94945) [reply config/routes.rb]: yeah should just do it all as a separate commit

File: config/routes.rb (hunk 1 above)
Comment (alex94945) [reply config/routes.rb]: meh, tried 2 and couldn't immediately get it to work. not sure what it buys w/ a single static page so whatevs for now

**Pull request merged**